module1/lessons/ruby_object_model_poly.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Object-Oriented Programming
+length: 60min
+layout: page
+---
+
+## Learning Goals
+
+* Understand and explain the Look Up Chain and how it relates to the principle of Polymorphism in Object-Oriented Programming.
+* Explore how Ruby's object model facilitates this principle.
+
+## Vocabulary
+
+* Inheritance
+* Polymorphism
+* Superclass
+* Module
+* Object Model
+* Look Up Chain
+
+## WarmUp
+
+* What is class inheritance and how is it implemented in Ruby?
+* What is a module in Ruby and how is it implemented?
+* How do you know what variables, methods, and classes you have available at any given time in Ruby?
+
+# The Ruby Object Model
+
+## Investigative Methods
+
+These three methods can help you investigate the relationships between classes and modules in Ruby. All methods are run on the class (i.e., `String`, `Hash`):
```

Comment:
Consider changing  (i.e., `String`, `Hash`) to (i.e., `String.ancestors`, `Hash.included_modules` ): just for more clarity on what running the method on a class looks like

Reply:
Good call--added!